crypto.go

```diff
@@ -59,8 +140,8 @@ func newSecret(cipherCode Cipher) ([]byte, error) {
 	return secret, nil
 }
 
-func newIV(cipherCode Cipher) ([]byte, error) {
-	iv := make([]byte, cipherCode.ivSize())
+func newIV(size int) ([]byte, error) {
```

Comment:
We have to use crypto/rand because these need to be cryptographically secure. Per comment from math/rand package, `For random numbers suitable for security-sensitive work, see the crypto/rand package.`